lib/xclarity_client/services/xclarity_service.rb

```diff
@@ -34,14 +34,15 @@ def get_object(uuids, includeAttributes, excludeAttributes)
         $lxca_log.info "XclarityClient::Endpoints::XClarityService get_object",
                        "Sending request to #{managed_resource} resource"
 
-        uuids.reject! { |uuid| UUID.validate(uuid).nil? } unless uuids.nil?
+        uuid_array = uuids.reject(&:empty?) unless uuids.nil?
```

Comment:
I think that would be good test if the element is also `nil`, once that if an array like `[nil, ""]` be passed as param, it will trigger an error, because `nil#empty?` not exist.

Reply:
@douglasgabriel The `blank?` condition doesn't already covers the checks for `nil`, `empty`, `[]` and `{}`?

Reply:
Sadly the `blank?` method is not available here